queries/rbs/highlights.scm

```diff
@@ -67,9 +63,9 @@
 (const_name (constant) @constant)
 (global_name) @property
 
-(parameter (var_name) @variable.parameter)
+(parameter (var_name) @parameter)
 
-(keyword) @variable.parameter
+(keyword) @parameter
```

Comment:
I think having a comment for this case is kindof needed, just to clear away the misunderstandings

Something like `keyword arguments` as a comment right on top of this capture will be nice

Reply:
You are right. I have added comments.